script.js

```diff
@@ -0,0 +1,108 @@
+const inputText = document.querySelector('#todo-input');
+const addTodoBtn = document.querySelector('#add-todo-btn');
+const todoListTitle = document.querySelector('#todo-list-title');
+const doneListTitle = document.querySelector('#done-list-title');
+const todoList = document.querySelector('.todo-list');
+const doneList = document.querySelector('.done-list');
+
+let items = [];
+
+// 입력한 할 일을 items에 추가
+const addNewTodo = () => {
+  const todoObject = {
+    id: null,
+    text: inputText.value,
+    isDone: false,
+  };
+
+  // 공백 입력, 중복 입력 방지
+  const index = items.findIndex((todo) => todo.text == inputText.value);
+  if (inputText.value && index === -1) {
+    items.push(todoObject);
+    render();
+  }
+
+  // 할 일 추가 후 인풋 값 비우기
+  inputText.value = '';
+};
+
+addTodoBtn.addEventListener('click', addNewTodo);
+
+// 엔터를 눌러도 입력되도록
+const enterKey = () => {
+  if (window.event.keyCode == 13) {
+    addNewTodo();
+  }
+};
```

Comment:
보통 함수의 이름은 
- 함수가 하는 일을 구체적이고 명확하게,
- 그리고 동사로 시작하게 지어주시면 좋습니다.

코드를 보았을때, todoInput에  keyup이벤트를 핸들링하는 함수인 것 같습니다.
그러면 다음과 같은 이름은 어떨까요?
```suggestion
const handleTodoInputKeyup = () => {
  if (window.event.keyCode == 13) { // keyCode 13 is ENTER
    addNewTodo();
  }
};
```

혹은 다음과 같이 `addEventListener`를 통해서 작성해보는 것도 좋은 것 같습니다!
```javascript
todoInput.addEventListener('keyup', () => { /* some event handling */ });
```

Reply:
네 수정하겠습니다! 감사합니다~!